includes/utils.hpp

```diff
@@ -18,14 +18,20 @@
 #include <string>
 #include <vector>
 
+#include "../includes/Server.hpp"
+
+typedef std::vector<std::string> stringVector;
+
 void shrink_to_fit(std::vector<struct pollfd> *vec);
 
-bool isNumeric(const std::string& str);
+bool isNumeric(const std::string &str);
 
 std::string trimWhiteSpaces(const std::string &str);
 
+void splitByComma(const std::string &argument, stringVector *channels);
```

Comment:
I've updated the name to `void splitByCommaAndTrim(const std::string &argument, stringVector *args)` to describe better its works.